src/store/lokapi.ts

```diff
@@ -120,7 +128,7 @@ export function lokapiStoreFactory(lokApiService: any) {
         try {
           commit('storeBackends', await lokApiService.getBackends())
         } catch (err:any) {
-          console.error('Error getting currency backends', err)
+          console.error('Error getting currency backends', err.message, "\nDebug: " + JSON.stringify(err))
```

Comment:
The arguments of console.error and console.log are stringified, no need to do it yourself and can be displayed by browser in a much better way by themselves (indeed, it is the reference of the object that is sent, so the exception gets to be dynamically printed and choose how it wants to be displayed. Maybe err.message is not used as you think it is in these exceptions). You are effectively casting the exception towards strings, and I don't understand the call to  `JSON.stringify()` here, I'm obviously missing something you wanted to achieve.

Reply:
I don't like my js console cluttered with stack traces, that's why I chose to make it more tidy. Have the reason of the error on the first line, then additional information below, as a string which can be copy-pasted by anyone, and give to someone competent to analyze. Maybe that's useless.

So if I understand you, you would just keep the `console.error('Error getting currency backends', err)` ?

Reply:
?
Changed.